Add ‘v1’ assets

36degrees opened the pull request:
Add the assets from the ‘Master Brand Assets_25062025’ folder in Google Drive.

Compared to the version in Google Drive:

- simplify the name of the top level folders
- exclude `All Motion_Master_NOT for v.1 release` as it’s not intended to be included at this time
- move all logo files up a level rather than having a separate folder per file type
- rename files in Logo/Crown/Black from `__Crown_redrawn.*` to `Crown_Black.*`
- rename files in Logo/Crown/Black from `__Crown_White_redrawn.*`  to `Crown_White.*`

Add a README to give an overview of the different files in the repo.

36degrees committed:
Add ‘v1’ assets

Add the assets from the ‘Master Brand Assets_25062025’ folder in Google Drive.

Compared to the version in Google Drive:

- simplify the name of the top level folders
- exclude `All Motion_Master_NOT for v.1 release` as it’s not intended to be included at this time
- move all logo files up a level rather than having a separate folder per file type
- rename files in Logo/Crown/Black from `__Crown_redrawn.*` to `Crown_Black.*`
- rename files in Logo/Crown/Black from `__Crown_White_redrawn.*`  to `Crown_White.*`

Add a README to give an overview of the different files in the repo.

querkmachine commented on Logo/Lockup/30px Web Header/Black/Web_30px_Header_Wordmark_Black.ai:
I'm not sure having the '30px web' versions of the files in here is valuable, and is more likely to confuse users as to what they're intended for.

I can't immediately see a difference between them and the 'large' lockup versions either, other than having a different viewbox size in the Illustrator files.

querkmachine commented on README.md:
```diff
@@ -1 +1,33 @@
+
+All logo versions are available in Adobe Illustrator (`.ai`), Encapsulated PostScript (`.eps`), Portable Network Graphic (`.png`) or Scalable Vector Graphic (`.svg`) format.
+
+### [Crown](Logo/Crown)
```
Suggest re-ordering these to wordmark, then crown, then lockup, as that's the hierarchy of use in the new brand guidelines.

querkmachine replied:
Could add a fixed height to each of the examples so they're proportional to one another rather than being massive?

36degrees replied:
> Suggest re-ordering these to wordmark, then crown, then lockup, as that's the hierarchy of use in the new brand guidelines.

Good shout 👍🏻 

> Could add a fixed height to each of the examples so they're proportional to one another rather than being massive?

I had a quick look but couldn't find a way to do that with the `<picture>` element – are you aware of a way to do it?

querkmachine replied:
Typically I would set the dimensions on the `<img>` element and it'd use those. Not sure if GitHub markdown changes how that works, though.

36degrees replied:
Yep, that works ✨ Should have tried the simple thing!

querkmachine commented on Logo/Lockup/Large/Primary Black/Web_Wordmark_PrimaryBlack.png:
Random noticing that the PNG versions of lockup and wordmark versions seem to have less whitespace than the SVG versions. Seems unexpected?

querkmachine reviewed (approved):
I think we can get rid of the 30px versions, and the other stuff is not technically blocking, but seem a bit weird or inconsistent.

36degrees committed:
Remove ‘Web 30px Header’ versions of lockup

We don’t think these are useful, so let’s remove them. We can re-add them later if we see demand for them.

36degrees committed:
Re-order logos in README to match guidelines

36degrees committed:
Add heights to images in README

querkmachine commented on Logo/Lockup/Black/Web_Wordmark_Black.ai:
I wonder if it's worth renaming "Web_Wordmark" to "Lockup" to match the folder name and to avoid confusion with the plain wordmark.

36degrees replied:
Good call 👍🏻

querkmachine reviewed (approved):
One more suggestion but good to go otherwise, I think!

36degrees committed:
Rename "Web_Wordmark" assets to "Lockup"

Match the folder name and avoid confusion with the plain wordmark.

querkmachine commented on README.md:
```diff
@@ -1 +1,34 @@
+  <img alt="Crown" src="Logo/Crown/Black/Crown_Black.svg" height="150">
+</picture>
+
+### [Lockup](Logo/Lockup)
```
URL paths in this section need updating for the new folder and file names.

36degrees replied:
🤦🏻
